_ml-commons-plugin/extensibility/index.md

```diff
@@ -0,0 +1,97 @@
+
+To get started with ML extensibility, choose from the following options.
+
+- If you're an ML developer wanting to integrate into your specific ML services, see [Building blueprints]({{site.url}}{{site.baseurl}}/ml-commons-plugin/extensibility/blueprints/).
+- If you're a sysadmin or data scientist wanting to create connection to a ML service, see [Creating connectors for third-party ML platforms]({{site.url}}{{site.baseurl}}/ml-commons-plugin/extensibility/connectors/).
+
+## Prerequisites
+
+If you are an admin deploying an ML connector, make sure that the target model of the connector has already been deployed on your chosen platform. Furthermore, make sure that you have permissions to send and receive data to the third-party API for your connector. 
+
+When access control is enabled on your third-party platform, you can enter your security settings using the `authorization` or `credential` settings inside the connector API.
+
+### Adding trusted endpoints
+
+To configure connectors in OpenSearch, add the trusted endpoints to your cluster settings using the `plugins.ml_commons.trusted_connector_endpoints_regex` setting, which supports Java regex expressions, as shown in the following example:
+
+```json
+PUT /_cluster/settings
+{
+    "persistent": {
+        "plugins.ml_commons.trusted_connector_endpoints_regex": [
+          "^https://runtime\\.sagemaker\\..*[a-z0-9-]\\.amazonaws\\.com/.*$",
+          "^https://api\\.openai\\.com/.*$",
+          "^https://api\\.cohere\\.ai/.*$"
+        ]
+    }
+}
+```
+{% include copy-curl.html %}
+
+
+
+### Setting up connector access control
+
+If you plan on using a remote connector, make sure to use an OpenSearch cluster with the Security plugin enabled. Using the Security plugin gives you access to connector access control, which is required when using a remote connector.
+{: .warning}
+
+If you require granular access control on your connectors, use the following cluster setting:
+
+```json
+PUT /_cluster/settings
+{
+    "persistent": {
+        "plugins.ml_commons.connector_access_control_enabled": true
+    }
+}
+```
+{% include copy-curl.html %}
+
+When enabled, you can install the [Security plugin]({{site.url}}{{site.baseurl}}/security/index/) the `backend_roles`, `add_all_backend_roles`, or `access_model` options are required in order to use the connector API. If successful, OpenSearch returns the following response:
+
+```json
+{
+  "acknowledged": true,
+  "persistent": {
+    "plugins": {
+      "ml_commons": {
+        "connector_access_control_enabled": "true"
+      }
+    }
+  },
+  "transient": {}
+}
+```
+
+### Node settings
+
+Remote models based on external connectors consume less resources. Therefore, you can deploy any model from a standalone connector using data nodes. To make sure that your standalone connection uses data nodes, set `plugins.ml_commons.only_run_on_ml_node` to `false`, as shown in the following example:
```

Comment:
Use "fewer" if the noun is countable and "less" if it isn't. Fewer cars, less traffic.